src/software/proto/message_translation/ssl_referee_test.cpp

```diff
@@ -2,4 +2,157 @@
 
 #include <gtest/gtest.h>
 
-// TODO (Issue #1458): Add tests
+#include "software/proto/ssl_referee.pb.h"
+#include "software/sensor_fusion/refbox_data.h"
+#include "software/world/team_colour.h"
+
+class RefboxStageTest
+    : public ::testing::TestWithParam<std::tuple<RefboxStage, SSL_Referee_Stage>>
+{
+};
+
+TEST_P(RefboxStageTest, test_refbox_stage)
+{
+    RefboxStage expected    = std::get<0>(GetParam());
```

Comment:
Perhaps also check these enums have the same number of values (if possible, they may not have COUNT at the end or something)

Reply:
They don't :/ its an enum defined in a proto